repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-link-folder.yaml

```diff
@@ -0,0 +1,16 @@
+definitions:
```

Comment:
Could we get rid of `new-link-folder` template query as it became a compound type now ?